Packs/FeedCiscoSecureMalwareAnalytics/Integrations/FeedCiscoSecureMalwareAnalytics/FeedCiscoSecureMalwareAnalytics.py

```diff
@@ -0,0 +1,211 @@
+    """
+    fetch_date = demisto.getIntegrationContext().get('last_fetch_date')
+    if fetch_date:
+        fetch_date = arg_to_datetime(fetch_date)
+    else:
+        fetch_date = first_fetch_date
+
+    if fetch_date.date() >= fetch_date.now().date():
+        fetch_date = fetch_date.now()
+        fetch_time_stamp = ''
+    else:
+        fetch_time_stamp = f'_{str(fetch_date.date())}'
+    indicators = []
+    temp = {}
+    if 'all' in feed_name:
+        feed_name = ['autorun-registry', 'banking-dns', 'dga-dns', 'dll-hijacking-dns', 'doc-net-com-dns',
+                     'downloaded-pe-dns', 'dynamic-dns', 'irc-dns', 'modified-hosts-dns', 'parked-dns',
+                     'public-ip-check-dns', 'ransomware-dns', 'rat-dns', 'scheduled-tasks', 'sinkholed-ip-dns',
+                     'stolen-cert-dns']
+    for name in feed_name:
+        result = client.get_indicators(feed_name=name, time_stamp=fetch_time_stamp)
+        for obj in result:
+            feed_related_indicators = [{'type': 'IP', 'value': ip} for ip in obj.get('ips')]
+            feed_related_indicators.append({'type': 'File', 'value': obj.get('sample_md5')})
+            feed_related_indicators.append({'type': 'File', 'value': obj.get('sample_sha256')})
+            feed_related_indicators.append({'type': 'File', 'value': obj.get('sample_sha1')})
+
+            if obj.get('domain') in temp:
+                if name not in temp[obj.get('domain')]['fields']['Tags']:
+                    temp[obj.get('domain')]['fields']['Tags'].append(name)
+
+                temp[obj.get('domain')]['fields']['FeedRelatedIndicators'].extend(
+                    related for related in feed_related_indicators
+                    if related not in temp[obj.get('domain')]['fields']['FeedRelatedIndicators'])
+
+            else:
+                temp[obj.get('domain')] = {
+                    "value": obj.get('domain'),
+                    "type": 'Domain',
+                    "rawJSON": obj,
+                    "fields": {
+                        "Tags": [name],
+                        "reportedby": 'CiscoSMA',
+                        "FeedRelatedIndicators": feed_related_indicators,
+                        "Description": obj.get('description')
+                    }
+                }
+            if tlp_color:
+                temp[obj.get('domain')]['fields']['trafficlightprotocol'] = tlp_color
+            if feed_tags:
+                temp[obj.get('domain')]['fields']['trafficlightprotocol'] += feed_tags
+    for key in temp:
+        indicators.append(temp[key])
+    next_fatch = (dateparser.parse('tomorrow', settings={'RELATIVE_BASE': fetch_date})).isoformat()
+    if not is_test:
+        demisto.setIntegrationContext({'last_fetch_date': next_fatch})
+        demisto.debug(f'{len(indicators)} XSOAR Indicators were created.')
+    return indicators
+
+
+def get_indicators_command(client: Client, args: Dict[str, str]) -> CommandResults:
```

Comment:
This command should have the same logic as the `fetch_indicators`. Let's make this function call the `fetch_indicators` function. In order to prevent the integration context change when calling the get-indicators command (since it appears in the `fetch_indicators` function), we should create a new function fetch-indicators-**command** which will first call the fetch-indicators and then handle both:
- demisto.setIntegrationContext
- demisto.createIndicators

The `test_module` and get-indicators functions will call the fetch-indicators function, and from the main function in the case of `fetch_indicators` flow, you will call the `fetch_indicators_command`. This way you also won't need the `is_test` argument.